locales/en.default.schema.json

```diff
@@ -386,6 +386,18 @@
   },
   "sections": {
     "all": {
+      "animation": {
+        "label": "Animations",
```

Comment:
I wonder if we should use the key `label` here as this is used as a header `content` on the schema. We usually keep a similar structure to what's used in our schema to make mapping easier. However, I see that just below we're using a `section_padding_heading` which also doesn't map to anything, so, not a big deal right now 👀